packages/react-core/src/demos/AlertGroup.md

```diff
@@ -0,0 +1,20 @@
+---
+id: Alert group
+section: components
+---
+
+import { useEffect } from 'react';
+import BellIcon from '@patternfly/react-icons/dist/js/icons/bell-icon';
+import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
+import DashboardWrapper from './examples/DashboardWrapper';
+import DashboardHeader from './examples/DashboardHeader';
```

Comment:
This is probably fine, I just wanted to ask if these need to be imported here since they're also imported in [AlertGroupToastWithNotificationDrawer.tsx](https://github.com/patternfly/patternfly-react/blob/69e08e8b523b5fe28dcf817f7f9ca8e982cf6caf/packages/react-core/src/demos/examples/AlertGroup/AlertGroupToastWithNotificationDrawer.tsx)?

Reply:
They need to be imported also here, we would get `ReferenceError` otherwise